homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
+  }
+
+  function renderRepoInformation(repo, ul) {
+    const url = `https://api.github.com/repos/HackYourFuture/${repo}`;
+    fetchJSON(url, (err, repoInfos) => {
+      if (err) {
+        renderError(err);
+        return;
+      }
+
+      ul.innerHTML = ' ';
```

Comment:
Is does not make sense to create a list with a single list item. This is the HTML you generate:

```html
<ul class="repoInfoList">
  <li>
    <table class="infoTable">
      <tbody>
        <tr>
          <td class="explanations">Repository:</td>
          <td class="list-item">
            <a href="https://github.com/HackYourFuture/alumni" target="blank">alumni</a>
          </td>
        </tr>
        <tr>
          <td class="explanations">Description:</td>
          <td>Alumni Guides, everything you need to land a job</td>
        </tr>
        <tr>
          <td class="explanations">Forks:</td>
          <td>9</td>
        </tr>
        <tr>
          <td class="explanations">Updated:</td>
          <td>2019-06-21T11:38:21Z</td>
        </tr>
      </tbody>
    </table>
  </li>
</ul>
```

It would make more sense to create a `div` or `section` to serve as a container for the `table`.